types/unist/index.d.ts

```diff
@@ -30,15 +38,6 @@ export interface Node {
     position?: Position | undefined;
 }
 
-/**
- * Information associated by the ecosystem with the node.
- * Space is guaranteed to never be specified by unist or specifications
- * implementing unist.
- */
-export interface Data {
-    [key: string]: unknown;
-}
```

Comment:
People will use this in the wild, removing it is a breaking change, and what for?